content/posts/2020-10-08-cmake-w-skrocie/index.md

```diff
@@ -0,0 +1,42 @@
+﻿---
+title: "CMake w skrócie"
+subtitle: "Fragment lekcji z pierwszej edycji kursu C++ online"
+date: 2020-10-08T13:00:00+02:00
+lastmod: 2020-10-08T13:00:00+02:00
+draft: false
+author: ""
+authorLink: ""
+description: ""
+summary: ""
+url: "/post/cmake-w-skrocie"
+
+tags: ["c++", "kurs online", "narzedzia", "cmake"]
+categories: ["Post", "Kurs C++ online", "W skrócie"]
+keywords: ["cmake poradnik", "cmake co to jest", "cmake jak używać"]
+hiddenFromHomePage: false
+hiddenFromSearch: false
+
+featuredImage: "featured.png"
+featuredImagePreview: "/post/cmake-w-skrocie/featured.png"
+
+toc:
+  enable: false
+math:
+  enable: false
+lightgallery: false
+share:
+  enable: true
+comment:
+  enable: true
+license: ""
+---
+
+# Make - narzędzia programisty
+
+Obecnie nikt nie wyobraża sobie kompilowania dużych projektów programistycznych bez systemu budowania.
+
+Dzięki Make możemy zautomatyzować proces kompilacji plików źródłowych i ich zależności. Możemy dzięki temu przyspieszyć kompilację, bo nie musimy przebudowywać plików obiektowych, które nie są zależne od naszych zmian. Aby jednak to osiągnąć, należy wiedzieć czym są pliki Makefile i jak ich używać.
+
+Jeśli nie wiesz jak tworzyć pliki Makefile, usiądź wygodnie i obejrzyj ten krótki film o Make podczas swojej przerwy na kawę ☕.
+
+{{< youtube j3N-xZlOc98 >}}
```

Comment:
to jest opis z make, nie z cmake

Reply:
Wstawiłem zły opis na asanie - teraz powinno być w porządku